README.md

```diff
@@ -95,6 +95,46 @@ Alternatively, the application can be started by using the provided Makefile:
 make start-all-heureka
 ```
 
+### Tests
+
+Heureka uses [Ginkgo](https://onsi.github.io/ginkgo/) for behavior-driven development (BDD) style tests. In the current project setup, tests are organized into three different directories, each serving a specific purpose:
+
+- End-to-End Tests: These tests are located in the ./internal/e2e directory. End-to-end tests are designed to test the flow of an application from start to finish and ensure all integrated pieces of an application function as expected together.
+
+- Application Layer Tests: These tests resides in the ./internal/app directory. Application layer tests focus on testing the application's behavior, such as handling requests and responses, executing appropriate logic, and more.
+
+- Database Tests: These tests are found in the ./internal/database/db directory. Database tests ensure that the application correctly interacts with the database. They test database queries, updates, deletions, and other related operations.
+
+If `LOCAL_TEST_DB` is set to `true` in `.test.env`, the tests will interact with a local database; otherwise, they will run against a containerized database.
```

Comment:
Maybe make clear the its exepcted that the local db already runs!

Reply:
Done